content/400-pulse/300-database-events.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: 'Database events'
+metaTitle: 'Database events with Prisma Pulse'
+metaDescription: 'Learn about database change events in Prisma Pulse, their structure, delivery semantics, and more.'
+tocDepth: 4
+toc: true
```

Comment:
I don't think we need these anymore. maybe [`toc_max_heading_level`](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#toc_max_heading_level) but [the toc is enabled by default](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#hide_table_of_contents) so you can remove `toc: true`.

```suggestion
```